recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -232,14 +245,54 @@ def startFirefox(self, width, height, env):
         options.add_argument(f'--width={width}')
         options.add_argument(f'--height={height}')
 
-        if disk_usage('/tmp').free < 134217728:
-            self._logger.warning('Less than 128 MiB available in "/tmp", strange failures may occur')
-
         service = FirefoxService(
             env=env,
         )
 
-        self.driver = webdriver.Firefox(
+        self.driver = FirefoxDriver(
+            options=options,
+            service=service,
+        )
+
+        self.bidiLogsHelper = BiDiLogsHelper(self.driver, self._parentLogger)
+
+    def startChromium(self, width, height, env):
```

Comment:
Could you move this code above `startFirefox()` for consistency with [_Talkbuchet-cli.py_](https://github.com/nextcloud/spreed/blob/eaee928da9aee60e567752908f3136b80b0157f3/docs/Talkbuchet-cli.py#L299-L344)?